pkg/controller/storagecluster/cephobjectstores.go

```diff
@@ -38,28 +38,35 @@ func (r *ReconcileStorageCluster) ensureCephObjectStores(instance *ocsv1.Storage
 }
 
 // createCephObjectStore creates CephObjectStore in the desired state
-func (r *ReconcileStorageCluster) createCephObjectStores(cephObjectStores []*cephv1.CephObjectStore, instance *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+func (r *ReconcileStorageCluster) createCephObjectStores(cephObjectStores []*cephv1.CephObjectStore, reqLogger logr.Logger) error {
 	for _, cephObjectStore := range cephObjectStores {
 		existing := cephv1.CephObjectStore{}
 		err := r.client.Get(context.TODO(), types.NamespacedName{Name: cephObjectStore.Name, Namespace: cephObjectStore.Namespace}, &existing)
 		switch {
 		case err == nil:
 			if existing.DeletionTimestamp != nil {
-				reqLogger.Info(fmt.Sprintf("Unable to restore init object because %s is marked for deletion", existing.Name))
-				return fmt.Errorf("failed to restore initialization object %s because it is marked for deletion", existing.Name)
+				err := fmt.Errorf("failed to restore initialization object %s because it is marked for deletion",
+					existing.Name)
+				reqLogger.Error(err,
+					fmt.Sprintf("Unable to restore init object because %s is marked for deletion", existing.Name))
+				return err
```

Comment:
Why are there so many error message changes? These should be in a separate commit. Also, please don't wrap the fmt/logging function lines.

Reply:
Corrected the error message and moved to the refactoring commit...